maths/liouville_lambda.py

```diff
@@ -0,0 +1,46 @@
+"""
+== Liouville Lambda Function ==
+The Liouville Lambda function, denoted by λ(n)
+and λ(n) is 1 if n is the product of an even number of prime numbers,
+and -1 if it is the product of an odd number of primes.
+
+https://en.wikipedia.org/wiki/Liouville_function
+"""
+
+# Author : Akshay Dubey (https://github.com/itsAkshayDubey)
+from maths.prime_factors import prime_factors
+
+
+def liouville_lambda(number: int) -> int:
+    """
+    # doctest: +NORMALIZE_WHITESPACE
```

Comment:
This isn't necessary here
```suggestion
```

Reply:
Fixed in [this](https://github.com/TheAlgorithms/Python/pull/7986/commits/8e110710d29e7d88478f349cbdfb96f3ef8c5870) commit. Kindly have a look.

Thanks